### PR TITLE
Wily metapackage

### PR DESCRIPTION
**Pull request by tonybaloney**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by tonybaloney** [comment]
@conda-forge/help-python I'm getting stuck with the linter here, it's saying 
>>> Non noarch: python packages should have a python requirement without any version constraints.
However, my package requires Python 3.6 and above?
I tried adding `noarch: python` but it has the same complaint.
If I remove the selector, then the build fails against Python 2.7.

What is the correct way to exclude Python 2.7 for the package?

File: recipes/wily/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  entry_points:
+    - wily = wily:main
+
+requirements:
+  host:
+    - python >=3.6
+    - pip >=19.1
+    - flit >=2.0
+  run:
+    - python
```
Comment (scopatz) [inline recipes/wily/meta.yaml]: You also need `>=3.6` here to restrict python 2

Comment (tonybaloney) [reply recipes/wily/meta.yaml]: @scopatz do I just ignore the linter when those restrictions are in place?
It gives this warning https://github.com/conda-forge/staged-recipes/pull/10416#issuecomment-564700699

**Comment by tonybaloney** [comment]
Also the windows build failure is because the build.bat seems to be copying the selectors into the file path:

```
OSError: [WinError 123] The filename, directory name, or volume label syntax is incorrect: 'C:\\Miniconda\\conda-bld\\wily_1576093548357\\work' -> 'C:\\Miniconda\\conda-bld\\wily_1576093548357\\work_moved_wily-1.15.0-py>=36_0_win-64_main_build_loop'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File ".ci_support\build_all.py", line 135, in <module>
    build_all(args.recipes_dir, args.arch)
  File ".ci_support\build_all.py", line 68, in build_all
    build_folders(recipes_dir, new_comp_folders, arch, channel_urls)
  File ".ci_support\build_all.py", line 125, in build_folders
    conda_build.api.build([recipe], config=get_config(arch, channel_urls))
  File "C:\Miniconda\lib\site-packages\conda_build\api.py", line 209, in build
    notest=notest, need_source_download=need_source_download, variants=variants)
  File "C:\Miniconda\lib\site-packages\conda_build\build.py", line 2409, in build_tree
    shutil.move(metadata.config.work_dir, dest)
  File "C:\Miniconda\lib\shutil.py", line 555, in move
    symlinks=True)
  File "C:\Miniconda\lib\shutil.py", line 315, in copytree
    os.makedirs(dst)
  File "C:\Miniconda\lib\os.py", line 220, in makedirs
    mkdir(name, mode)
OSError: [WinError 123] The filename, directory name, or volume label syntax is incorrect: 'C:\\Miniconda\\conda-bld\\wily_1576093548357\\work_moved_wily-1.15.0-py>=36_0_win-64_main_build_loop'
```

**Pull request closed without merging**